src/data/script_i18n.ts

```diff
@@ -46,6 +50,11 @@ export const describeAction = <T extends ActionType>(
     actionType = getActionType(action) as T;
   }
 
+  const conjunctionFormatter = new Intl.ListFormat(hass.language, {
+    style: "long",
+    type: "conjunction",
+  });
```

Comment:
We should start moving these to their own files, `src/common/string/format-list.ts`